jetstream/sponsored-suggestion-placement-phase-2.toml

```diff
@@ -0,0 +1,694 @@
+[experiment]
+
+enrollment_period = 14
+end_date = '2024-04-22'
+segments = ["cohort_1", "cohort_2"] # Two cohorts
+
+## Segments
+
+[segments]
+[segments.cohort_1]
+select_expression = "enrollment_date BETWEEN '2024-01-17' AND '2024-01-30'"
+data_source = "segment_enrollments"
+
+[segments.cohort_2]
+select_expression = "enrollment_date BETWEEN '2024-01-17' AND '2024-02-06'"
+data_source = "segment_enrollments"
+
+[segments.data_sources.segment_enrollments]
+client_id_column = "client_id"
+submission_date_column = "enrollment_date"
+friendly_name = "Enrolment table used to calculate above segments"
+from_expression = """(
+    SELECT
+        e.client_id,
+        `mozfun.map.get_key`(e.event_map_values, 'branch')
+            AS branch,
+        MIN(e.submission_date) AS enrollment_date,
```

Comment:
We'll need to rename `enrollment_date` here because the actual enrollment date will show in the queries and now we have two columns named the same. How about something like `segment_enrollment_date`